packages/eas-cli/src/graphql/mutations/PublishMutation.ts

```diff
@@ -31,7 +31,7 @@ const PublishMutation = {
 
   async publishUpdateGroupAsync(
     publishUpdateGroupInput: PublishUpdateGroupInput
-  ): Promise<Pick<Update, 'id' | 'group'>> {
+  ): Promise<Pick<Update, 'group'>> {
```

Comment:
We should encourage the use of `group` instead of `id` when dealing with updates.